scripts/test-cache-manager.sh

```
@@ -0,0 +1,277 @@
+    
+    # Results cache
+    local results_count=$(find "$CACHE_DIR/results" -name "*.tap" 2>/dev/null | wc -l)
+    echo "Cached test results: $results_count"
+    
+    # Docker cache
+    local docker_count=$(find "$CACHE_DIR/docker" -name "*.tar.gz" 2>/dev/null | wc -l)
+    echo "Cached Docker images: $docker_count"
+    
+    # Age of oldest cache entry
+    if [[ $results_count -gt 0 ]]; then
+        local oldest=$(find "$CACHE_DIR/results" -name "*.tap" -exec stat -f %m {} \; 2>/dev/null | \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)